topics/spacevim/index.md

```diff
@@ -0,0 +1,16 @@
+---
+aliases: space vim
+created_by: ShiDong Wang
```

Comment:
On http://spacevim.org/about/, the name is displayed as "Wang Shidong".

Reply:
Will change the about page，My first name is ShiDong，Last name is Wang.

Reply:
Ok, sorry. I still prefer Wang Shidong. It is just same as my chinese name.

Reply:
Oh, sorry about that! I didn't realize you were the one who submitted the PR. 😅